content/blog/2017-11-28-fragments-in-react-v16.2.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "Fragments in React v16.2"
+author: [clemmy]
+---
+
+Today, we're excited to bring the new `Fragment` export to React, which allows users to render **static** fragments without the unnecessary keying of each element in the fragment!
```

Comment:
This paragraph is slightly confusing because it doesn't mention the new *syntax* but shows the new syntax. Maybe:

>Today, we’re excited to bring the new `Fragment` export to React, which allows users to render **static** fragments without the unnecessary keying of each element in the fragment!
>
>```js
>render() {
>  return (
>    <React.Fragment>
>      <li>Look ma</li>
>      <li>no keys!</li>
>    </React.Fragment>
>  );
>}
>```
>
>There is also a new short syntax for it that is already supported by [some tools](#how-to-use-it):
>
>```js
>render() {
>  return (
>    <>
>      <li>Look ma</li>
>      <li>no keys!</li>
>    </>
>  );
>}
>```